### PR TITLE
Add Rollbar configuration to iOS kickoff guide

### PR DESCRIPTION
**Pull request by pablog**

### CODE REVIEW

**Comment by danielaRiesgo** [comment]
@pablog reference issue #61 .

File: iOS/docs/kickoff/README.md [hunk 1]
```diff
@@ -146,6 +146,25 @@ And commit them:
 
 To configure continuous integration follow the guide [CI configuration for iOS projects](./configure-ci/README.md)
 
+### Configure Crash logger
+
+The project integrates [Rollbar](https://github.com/rollbar/rollbar-ios) as default crash logger.
+
+In order to enable it, ask a technical leader to create a new group and a new iOS project.
```
Comment (danielaRiesgo) [inline iOS/docs/kickoff/README.md]: `... and a new iOS project in Rollbar.`

File: iOS/docs/kickoff/README.md [hunk 2]
```diff
@@ -146,6 +146,25 @@ And commit them:
...
+
+Then copy into all the `.xcconfig` files the following content:
+
+```
+ROLLBAR_CLIENT_ACCESS_TOKEN = ...
+ROLLBAR_SERVER_ACCESS_TOKEN = ...
+```
+
+Where `ROLLBAR_CLIENT_ACCESS_TOKEN` is the token under "post_client_item", and `ROLLBAR_SERVER_ACCESS_TOKEN` is the token under "post_server_item".
+
```
Comment (danielaRiesgo) [inline iOS/docs/kickoff/README.md]: Another possibility is writing:
```
ROLLBAR_CLIENT_ACCESS_TOKEN = <token under "post_client_item">
ROLLBAR_SERVER_ACCESS_TOKEN = <token under "post_server_item">
```

Don't know what is better. Any is good for me.

File: iOS/docs/kickoff/README.md [hunk 3]
```diff
@@ -146,6 +146,25 @@ And commit them:
...
+Once the `.xcconfig` files are configured (make sure they are ignored, if they are shown as changes to be committed, then do `git rm --cached ./BaseProject/ConfigurationFiles/*` to ignore this changes) just uncomment the call to `RollbarService().initialize()` in `AppDelegate.swift`.
```
Comment (danielaRiesgo) [inline iOS/docs/kickoff/README.md]: I think the `git rm --cached` will delete the files. Maybe they should use `git update-index --assume-unchanged <file>` ? I really never could make it work as I wanted.

Comment (pablog) [inline iOS/docs/kickoff/README.md]: You're right!

File: iOS/docs/kickoff/README.md [hunk 4]
```diff
@@ -146,6 +146,23 @@ And commit them:
 
 To configure continuous integration follow the guide [CI configuration for iOS projects](./configure-ci/README.md)
 
+### Configure Crash logger
+
+The project integrates [Rollbar](https://github.com/rollbar/rollbar-ios) as default crash logger.
+
+In order to enable it, ask a technical leader to create a new group and a new iOS project in Rollbar.
+
+Once created, sign in to [Rollbar](https://rollbar.com), and go to your project "Settings"/"Access Tokens".
```
Comment (danielaRiesgo) [inline iOS/docs/kickoff/README.md]: `... to your project's "Settings" -> "Access Tokens".` ?

**Pull request merged**